nasa-gibs/worldview
# WV-2193: Use monospace font everywhere we show a date

## Description

* Update fonts everywhere a date is shown to be monospace

## How To Test

Check the following places for proper font use and any possible style disturbances from these change.  Only the places that were affected by these changes are listed as some other parts of the app were already using monospace fonts.

* Layer info in sidebar
  * Check layers with no dates, only start date, and both start/end dates
* Layer info in product picker (search & measurement browse layer descriptions)
* Event list (individual events and each date within multi-point event)
* Layer coverage panel
* Compare mode:
  * A/B tabs in sidebar
  * Dates on either side of swipe line
  * Dates on either side of opacity slider
  * Date on spy circle
* GIF capture dialog
* GIF results dialog


@nasa-gibs/worldview
